scripts/check-codestyle.js

```diff
@@ -11,6 +11,23 @@ const { logger, process } = require('./utilities');
 const exec = require('child_process').execSync;
 let exitCode = 0;
 
+const ignoreMemo = {};
+const isIgnored = (file) => {
+  if (file in ignoreMemo) {
+    return ignoreMemo[file];
+  }
+  try {
+    // If this returns zero, it means the file is ignored by git; skip it.
+    exec(`git check-ignore -q '${file}'`);
```

Comment:
Returns zero as in fails and we go in catch block? Trying to understand if we have to check the console logged value and the decide.

Reply:
0 in bash is actually passing :)

Reply:
Ok thanks, so when the status exits with 1 its an error and its catched, thanks for explanation 🙏